core/src/main/java/org/apache/calcite/adapter/enumerable/EnumUtils.java

```diff
@@ -796,4 +803,244 @@ static Expression windowSelector(
         outputPhysType.record(expressions),
         parameter);
   }
+
+  /**
+   * Create enumerable implementation that applies sessionization to elements from the input
+   * enumerator based on a specified key. Elements are windowed into sessions separated by
```

Comment:
`Create` -> `Creates`, same for other codes.

Reply:
Done.